index.html

```diff
@@ -0,0 +1,76 @@
+
+<!doctype html>
+<html>
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	
+	<link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+	<link  rel="stylesheet" type="text/css" href="style.css">
+	<title>Cubes</title>
+</head>
+<body>
+	<div class="wrap">
+	
+		<section class="main_content">
+
+			<div class="cards card1">
+				<img src="img\chicago.png" alt="chicago" />
+				<p>Chicago / 23°</p>
+			</div>
+
+			<div class="cards card2">
+				<img src="img\seatle.png" alt="seatle" />
+				<p>Seattle / 34°</p>
+			</div>
+
+			<div class="cards card3">
+				<img src="img\brooklyn.png" alt="brooklyn" />
+				<p>Brooklyn / 55°</p>
+			</div>
+
+			<div class="cards card4">
+				<img src="img\portland.png" alt="portland" />
+				<p>Portland / 27°<p>
+			</div>
+
+			<div class="cards card5">
+				<div class="card5-top">
+					<img src="img\submenu.png" alt="selected card" />
+				</div>
+				<div class="card5-bottom">
+					<div class="day">
+						<p>MON</p>
+						<img src="img\chicago.png" alt="monday" />
+						<p>21°</p>
+					</div>
+					<div class="day">
+						<p>TUE</p>
+						<img src="img\chicago.png" alt="tuesday" />
+						<p>19°</p>
+					</div>
+					<div class="day">
+						<p>WEN</p>
+						<img src="img\san jose.png" alt="wendsday" />
+						<p>16°</p>
+					</div>
+					<div class="day">
+						<p>THU</p>
+						<img src="img\brooklyn.png" alt="thursday" />
+						<p>14°</p>
+					</div>
+					<div class="clearfix"></div>
+				</div>
+			</div>
+
+			<div class="cards card6">
+				<img src="img\san jose.png" alt="sanjose" />
+				<p>San jose / 47°</p>
+			</div>
+
+			<div class="clearfix"></div>

```

Comment:
This tag is not needed, please remove it. It is not considered best practice to add tags solely for the purposes of styling. All styling shall be done on the existing (and explicitly needed for the layout) elements.

Reply:
Thank you so much for your report! Higly appreciated. Uploaded patch and fixed all adviced issues.